test/test_http.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+from conftest import have_json
+from helper import run
+
+
+def test_range(http_server):
+    http_server.setup(
+        file_path="test/good-verity-bundle.raucb",
+    )
+
+    resp = http_server.get(headers={"Range": "bytes=0-3"})
+    resp.raise_for_status()
+    assert resp.status_code == 206
+    assert resp.content == b"hsqs"
+
+    summary = http_server.get_summary()
+    assert summary["requests"] == 1
+    assert summary["range_requests"] == ["0:4"]
+
+
+def test_headers(http_server):
```

Comment:
This is more like a test-internal sanity check, right?

Reply:
I've added some comments.